File: android/app/src/main/java/com/masreferenceapp/Status.java
package com.masreferenceapp;

import androidx.annotation.NonNull;

import org.json.JSONObject;

public class Status {
    public static String status(String statusCode, String message){
        try {
            JSONObject jo = new JSONObject();
            jo.put("statusCode", statusCode);
            jo.put("message", message);
            return (jo.toString());
        } catch (Exception e){
            return e.toString();
        }
    }
}

File: android/app/src/main/java/com/masreferenceapp/Status.kt
package com.masreferenceapp

import org.json.JSONObject

object Status {
    @JvmStatic
    @Deprecated("Use the new ReturnStatus() class.")
    fun status(statusCode: String?, message: String?): String {
        return try {
            val jo = JSONObject()
            jo.put("statusCode", statusCode)
            jo.put("message", message)
            jo.toString()
        } catch (e: Exception) {
            e.toString()
        }
    }
}
